governance/veANGLE/gauges.md

```diff
@@ -17,23 +17,19 @@ Like what was done weekly on Snapshot by ANGLE holders before the Angle Governan
 
 A gauge corresponds in fact to a staking contract and by voting for a gauge a veANGLE owner increases the amount of ANGLE tokens sent to the related staking contract with respect to other staking contracts. veANGLE owners will therefore vote for the gauges they think should receive more weight.
 
-This allows veANGLE holders who are the most long term users of the protocol to have complete control over the future ANGLE emission rate.
+This allows veANGLE holders, who are the most long term users of the protocol, to have complete control over the future ANGLE emissions.
 
 ## ↔️Incentives alinement
 
 Interestingly, the gauge system lowers the influence of ANGLE pairs and pools where the majority of rewards are sold off since those LPs will not have veANGLE to continue voting for their pair.
 
-This system strongly favors LP providers who continually stake their rewards for veANGLE to increase their pool's gauge weight. Essentially, and like mentionned in [Frax docs](https://docs.frax.finance/vefxs/gauge), Angle gauges align incentives of veANGLE holders so that the most long term oriented ANGLE holders control where ANGLE emissions.
+This system strongly favors LP providers who continually lock their rewards into veANGLE to increase their pool's gauge weight. Essentially, and like mentionned in [Frax docs](https://docs.frax.finance/vefxs/gauge), Angle gauges align incentives of veANGLE holders so that the most long term oriented ANGLE holders control where ANGLE emissions.
 
 ## In Practice
 
 Gauge weights are updated once every week every Thursday at 2am CET. This means that the ANGLE emission rate for each pair is constant for 1 week then updates to the new rate on each Thursday morning. Any user can change their weight allocation for a given gauge every 10 days.
 
-Weights of a given user for a gauge is given in basis points (units of 0.01%), this means that a veANGLE owner has 10000 weights to allocate.
-
-For instance, a user could decide to allocate 5000 to a gauge and 5000 to another. This means that this person would allocate 50% of its voting power to each of the two gauges.
-
-A person could also decide not to allocate all its available voting power. 3000 weights could be given to a gauge, and leave 7000 unallocated.
+When updating their weights, users will dedicate a share of their total veANGLE balances towards each gauge. Then, the sum of all the veANGLE assigned to each gauge by all holders will determine the quantity of rewards to be distributed.
```

Comment:
It's important to keep the part about people deciding not to allocate all their voting power